examples/s3_access.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "metadata": {
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.8.5-final"
+  },
+  "orig_nbformat": 2,
+  "kernelspec": {
+   "name": "python3",
+   "display_name": "Python 3.8.5 64-bit",
+   "metadata": {
+    "interpreter": {
+     "hash": "31862cc71836a94b0e0781803a3648767fc4cb197cc35bade0ddf231ddce7d7c"
+    }
+   }
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "source": [
+    "## Harmony Py Library\n",
+    "### Job Results Example"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "sys.path.append('..')\n",
+    "from harmony import BBox, Client, Collection, Request\n"
+   ]
+  },
+  {
+   "source": [
+    "Cloud access credentials can be retrieved from an instantiated Client."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "harmony_client = Client()  # assumes .netrc usage\n",
+    "\n",
+    "creds = harmony_client.aws_credentials()\n"
+   ]
+  },
+  {
+   "source": [
+    "Then, instantiate a boto3 client with those credentials."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3\n",
+    "\n",
+    "s3 = boto3.client('s3', **creds)\n",
+    "uri = 's3://harmony-uat-staging/public/harmony/netcdf-to-zarr/817a3e99-d53c-4169-b9f1-82cc947793be/2020_01_01_7f00ff_global.zarr'\n",
```

Comment:
Note that this example will eventually expire and be gone.  I think your intent here is for this to be temporary until we do the ticket for getting S3 URLs.

If any prior examples translate to Zarr, you could cheat and reuse a result from there for this part of the notebook, since those results will always be S3 URLs.